cdc/sink/ddlsink/mq/ddlproducer/pulsar_ddl_producer_test.go

```diff
@@ -18,9 +18,9 @@ import (
 	"testing"
 
 	"github.com/pingcap/tiflow/cdc/model"
+	config2 "github.com/pingcap/tiflow/pkg/config"
```

Comment:
There is no need to alias this import to `config2`.

Reply:
ok